tests/acceptance/config/behat.yml

```diff
@@ -121,6 +121,14 @@ default:
         - FeatureContext: *common_feature_context_params
         - LoggingContext:
 
+    cliProvisioning:
+      paths:
+        - '%paths.base%/../features/cliProvisioning'
```

Comment:
It also needs a matrix entry in ``.drone.yml``
And I expect that the path here needs to be like above:
```
%paths.base%/../../../../../tests/acceptance/features/cliProvisioning
```
because it needs to find its way out of the app folders back to core.